src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -91,13 +91,6 @@ export class UBSOrderDetailsComponent extends FormBaseComponent implements OnIni
     return this.showTotal > 0;
   }
 
-  translateWords(key: string, variable) {
-    return this.translate
-      .get(key)
-      .pipe(take(1))
-      .subscribe((item) => (variable = item));
-  }
```

Comment:
Remove unused import of 'take' from this file

Reply:
done